.idea/git_toolbox_prj.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```

Comment:
Please add the `.idea` folder in your global .gitignore

Reply:
Commit a16dd42a463734703ec942a590344b35fc9fb3c4 got rid of it